src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -351,8 +351,8 @@
   (when-let [canonicalized-path (decode-and-canonicalize-path path)]
      ;; Here, keywords represent a single element in the path. Anything between two '/' counts.
      ;; The second vector takes anything else that might be on the end of the path.
-     ;; Below, this corresponds to 'modules/*/files/**' in a filesystem glob.
-     (bidi.bidi/match-route [["modules/" :module-name "/files/" [#".*" :rest]] :_]
+     ;; Below, this corresponds to '*/*/files/**' in a filesystem glob.
+     (bidi.bidi/match-route [[#"[^/]+/" :module-name "/files/" [#".*" :rest]] :_]
```

Comment:
An alternative here would be `["/" :modules-dir "/" :module-name "/files/" [#".*" :rest]]`, but I'm not sure if it's cleaner.